### PR TITLE
Various touch-ups to Python docs

### PR DESCRIPTION
**Pull request by brettcannon**

### CODE REVIEW

File: docs/python/python-tutorial.md [hunk 1]
```diff
@@ -47,13 +47,13 @@ IntelliSense and auto-completions work for standard Python modules as well as ot
...
 
-Feel free to experiment with IntelliSense some more, but then revert changes so you have only the `msg` variable and the `print` statement, and save the file (`kb(workbench.action.files.save)`).
+Feel free to experiment with IntelliSense some more, but then revert changes so you have only the `msg` variable and the `print` call, and save the file (`kb(workbench.action.files.save)`).
 
 For full details on editing, formatting, and refactoring, see [Editing code](/docs/python/editing.md). The Python extension also has full support for [Linting](/docs/python/linting.md).
 
 ## Running Hello World
 
-It's simple to run `hello.py` with Python. From an external terminal, just type `python hello.py`, and you should see "Hello World" as output.
+It's simple to run `hello.py` with Python. From an external terminal, just type `python3 hello.py`, and you should see "Hello World" as output.
```
Comment (kraigb) [inline docs/python/python-tutorial.md]: Why "python3" when the python paths in the settings just use "python"? In my 3.6 installation, it's just "python.exe" not python3.exe:

c:\anaconda3\python.exe
C:\python36-32\python.exe

We could say "(or `python3 hello.py` depending on your installation)" if you're concerned about it.

File: docs/python/settings-reference.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ MetaSocialImage: TBD
 
 | Setting | Default | Description |
 | --- | --- | --- |
-| python.pythonPath | `"Python"` | Path to the python interpreter. |
-| python.venvPath | `""` | Path to folder with a list of Virtual Environments, such as ~/.pyenv, ~/Envs, ~/.virtualenvs. |
+| python.pythonPath | `"python"` | Path to the python interpreter. |
```
Comment (kraigb) [inline docs/python/settings-reference.md]: Here's an example where it's not python3, just python.

File: docs/python/unit-testing.md [hunk 3]
```diff
@@ -70,20 +70,20 @@ You can also configure pytest using a `pytest.ini` file as described on [PyTest
...
 
 
 ## Test discovery
 
-VS Code uses the currently enabled unit testing framework in to discover tests by applying the respective discovery patterns specified in the arguments option for that framework.
+VS Code uses the currently enabled unit testing framework to discover tests by applying the respective discovery patterns specified in the arguments option for that framework.
 
-For example, the default arguments for unittest include `-s . -p *_test.py`, meaning that unittest looks recursively, starting with the project folder (`-s .`), for all files with names matching the "*_test.py" pattern (`-p *_test.py`). You can specify a different starting folder after `-s`, and/or a different pattern after `-p`.
+For example, the default arguments for unittest include `-s . -p test*.py`, meaning that unittest looks recursively, starting with the project folder (`-s .`), for all files with names matching the "*_test.py" pattern (`-p *_test.py`). You can specify a different starting folder after `-s`, and/or a different pattern after `-p`.
```
Comment (kraigb) [inline docs/python/unit-testing.md]: The pattern is actually "*test*.py" (not sure where I got the _). Can you then edit the other instances of _test here to be *test*?

File: docs/python/debugging.md [hunk 4]
```diff
@@ -170,11 +170,10 @@ Remote debugging allows you to step through a program locally within VS Code whi
 
 Windows:
 1. Enable ssh port forwarding on the remote computer using sshd_config or similar.
-2. Establish a Putty SSH tunnel:
-    <ol>
-    <li><a href="http://realprogrammers.com/how_to/set_up_an_ssh_tunnel_with_putty.html">http://realprogrammers.com/how_to/set_up_an_ssh_tunnel_with_putty.html</a> (until Open the session section).</li>
-    <li>On the Tunnels screen, using a local mode, source port (which is the port which will be the entry point on the local computer) can be different from the destination port (the end point on the server).</li>
-    <li>Destination address should be the localhost or 127.0.0.1 address (which is the address that the remote ssh server will use to establish the tunnel).</li></ol>
+2. Establish a PuTTY SSH tunnel:
+    1. http://realprogrammers.com/how_to/set_up_an_ssh_tunnel_with_putty.html (until "Open the session" section).
```
Comment (kraigb) [inline docs/python/debugging.md]: Add the link back.

File: docs/python/environments.md [hunk 5]
```diff
@@ -21,19 +21,21 @@ This command automatically looks for and displays a list of available Python int
 
 ![List of interpreters](images/environments/interpreters-list.png)
 
-Selecting an interpreter from the list configures the `settings.json` file accordingly.
+Selecting an interpreter from the list configures the `settings.json` file accordingly. The statusbar will also be updated to show the selected interpreter, which can also be clicked on to bring up the list of available Python interpreters. If no interpreter is selected, the statusbar will reflect this fact:
```
Comment (kraigb) [inline docs/python/environments.md]: Edit out the "will" words:  "The statusbar also shows the selected interpreter, which brings up a list of available interpreters when selected. The statusbar also reflects when no interpreter is selected."

File: docs/python/linting.md [hunk 6]
```diff
@@ -52,7 +52,7 @@ The sections that follow provide additional details for those individual linters
 
 #### Command-line arguments and configuration files
 
-See [Pylint command line arguments](https://pylint.readthedocs.io/en/latest/user_guide/run.html#command-line-options) for general switches. Command line arguments can be used to load Pylint plugins, such that that for Django:
+See [Pylint command line arguments](https://pylint.readthedocs.io/en/latest/user_guide/run.html#command-line-options) for general switches. Command line arguments can be used to load Pylint plugins, such that for Django:
```
Comment (kraigb) [inline docs/python/linting.md]: Edit at end, "such as that for Django" (add "as")

File: docs/python/linting.md [hunk 7]
```diff
@@ -112,7 +112,7 @@ The Python extension maps pep8 message categories to VS Code categories through
 
 Flake8 user options are read from the `.flake8` (Windows) or `~/.config/flake8` (Mac/Linux) file.
 
-At the project level, options are read from the `[flake8]` section of a `tox.ini` or `setup.cfg` file. Only the first file is considered. For details, seer [Flake8 Configuration](http://flake8.readthedocs.org/en/latest/config.html).
+At the project level, options are read from the `[flake8]` section of a `tox.ini` or `setup.cfg` file. Only the first file is considered. For details, seef [flake8 Configuration](http://flake8.readthedocs.org/en/latest/config.html).
```
Comment (kraigb) [inline docs/python/linting.md]: "seef [flake8" >>"see [flake8" (remove the 'f')

File: docs/python/debugging.md (hunk 4 above)
Comment (brettcannon) [reply docs/python/debugging.md]: Markdown auto-links URLs. Does the VS Code site not follow that?

File: docs/python/python-tutorial.md (hunk 1 above)
Comment (brettcannon) [reply docs/python/python-tutorial.md]: The `python`-only thing is Windows only. On macOS and Linux the only way to have `python` point to Python 3 is to manually tweak it or to be using a virtual environment.

File: docs/python/debugging.md (hunk 4 above)
Comment (kraigb) [reply docs/python/debugging.md]: I didn't see it doing that on the local server, which is why I had the link there. @gregvanl do you know for certain?

File: docs/python/unit-testing.md (hunk 3 above)
Comment (brettcannon) [reply docs/python/unit-testing.md]: Did you mean `test.py` or `*test*`.py?

File: docs/python/python-tutorial.md (hunk 1 above)
Comment (kraigb) [reply docs/python/python-tutorial.md]: Does that not affect settings like pythonPath then? What I'm concerned about is that if you put in 'python3' then it breaks for Windows just as "python" breaks for Linux. So I think we need both if they are different.

File: docs/python/unit-testing.md (hunk 3 above)
Comment (kraigb) [reply docs/python/unit-testing.md]: it's `*test*.py` in the settings. GitHub comments italicized it for me without the backticks. :)

File: docs/python/python-tutorial.md (hunk 1 above)
Comment (brettcannon) [reply docs/python/python-tutorial.md]: I should also mention that `python` is not on your PATH on Windows unless you opt into it being there.

File: docs/python/settings-reference.md (hunk 2 above)
Comment (brettcannon) [reply docs/python/settings-reference.md]: That's to support Windows and its lack of `python3` command.

**Comment by brettcannon** [comment]
I've addressed your reviewer comments, @kraigb . PTAL.

**Pull request merged**